index.html

```diff
@@ -17,10 +17,27 @@
   </head>
   <body>
     <header>
-      <div class="container"></div>
+      <div class="container">
+        <h1>Code Journal</h1>
+      </div>
     </header>
     <main class="container">
-      <div data-view="entry-form"></div>
+      <div data-view="entry-form">
+        <div class="row">
+          <div class="column-full" id="column-half">
+            <form>
+              <h2>New Entry</h2>
+              <input type="image" id="img"src="./images/placeholder-image-square.jpg" alt="placeholder"><br>
+              <label for="title">Title</label><br>
+              <input type="text" id="titlerow"required><br>
+              <label for="PhotoURL" id="photolink">Photo URL</label><br>
+              <input type="text" id="photorow" required><br>
+              <label for="notes">Notes</label><br>
+              <input type="text" id="notesrow"required><br>
```

Comment:
This last input and label should be wrapped in it's own `row` and `column-full` divs. So that it takes up the full container under the photo image, title, and photo URL inputs above it.